dev/source/docs/learning-ardupilot-the-example-sketches.rst

```diff
@@ -53,6 +53,29 @@ connect to a Pixhawk on Linux:
 Using the --setup option puts mavproxy into raw serial mode, instead of
 processed MAVLink mode. That is what you need for the example sketches.
 
+Running sketches in SITL
+========================
+
+Certain sketches can also be run in SITL. For example to run the protocol decoder sketch:
+
+::
+
+    cd $ARDUPILOT_HOME # the top-level of an AruPilot repository
+    ./waf configure --board sitl
+    ./waf build --target examples/RCProtocolDecoder
+    
+To start the sketch, run it directly:
+
+::
+
+     ./build/sitl/examples/RCProtocolDecoder -M quad
+     
+In another window connect to it via mavproxy:
+
+::
+
+    mavproxy.py --setup --master tcp:127.0.0.1:5760 --console
```

Comment:
Do you need setup parameter?

Reply:
The rest of the page says yes, and that works for me - so I think yes

Reply:
ha my bad ! No worries!